bodhi/tests/server/test___init__.py

```diff
@@ -190,6 +190,8 @@ def test_authtkt_timeout_defined(self, set_authorization_policy, set_authenticat
         assert policy.cookie.secure == True
         assert policy.cookie.secret == 'hunter2'
         assert policy.cookie.timeout == 10
+        assert policy.cookie.samesite == "Strict"
```

Comment:
Apparently this is failing with `
E       AttributeError: 'AuthTktCookieHelper' object has no attribute 'samesite'
`